README.md

```diff
@@ -4,9 +4,11 @@
 
 This is an event page to provide information on and promote an upcoming yoga retreat.
 
-<img width="710" alt="Screenshot_mobile_portfolio" src="#">
+<img width="350" alt="Screenshot_mobile" src="images/mobile_screenshot.png">
 
-The website consists of a "home" and an "about" page. 
+<img width="710" alt="Screenshot_desktop" src="images/desktop_screenshot.png">
+
+The website consists of a "home" and an "about" page.
 
```

Comment:
- [ ] Please work on the `MORE` button, if a user clicks on it to show the remaining sections when it comes to the mobile view. 

![htmll11](https://user-images.githubusercontent.com/55635977/142261648-672a21bb-c5bf-4c63-be04-12abcdc08f33.png)

Reply:
I have also implemented the "More" button: 
Before pressing the button: 
<img width="486" alt="Screen Shot 2021-11-17 at 16 34 47" src="https://user-images.githubusercontent.com/75140420/142293409-9e347ed4-1121-4eed-90f3-542147c57f13.png">
After: 
<img width="480" alt="Screen Shot 2021-11-17 at 16 34 54" src="https://user-images.githubusercontent.com/75140420/142293454-0eeb697e-1cd8-40d6-937f-c9419256a956.png">